### ISSUE TITLE
Can't change the label of a layer

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#833]
When you load a layer from a remote URL via the `<layer- src>` attribute, it's expected that you won't have control over the accessible name of the layer - it's set via the `<map-title>` element in the remote document.  However, if there's no `<map-title>` element in the document, you should be able to set the `label` attribute of the `<layer->` element and see that change reflected in the layer control, but it not currently that way.  If the `<layer->` has inline content, i.e. no `src` attribute, you should be able to control the accessible name of the layer in the layer control by adding/changing the `<map-title>` child element of `<layer->`, or maybe the `<layer- label>` value as well tb discussed.

Note that [changing the accessible name of a thing is not advisable](https://sarahmhigley.com/writing/playing-with-state/#name-changes).

**Comment by prushforth** [comment]
The behaviour of HTMLLayerElement.label is not yet [specified](https://github.com/Maps4HTML/MapML/issues/246), but it is noted in the docs that [this needs implementation](https://maps4html.org/web-map-doc/docs/api/layer-api/#src:~:text=HTMLLayerElement.label%20provides).
